libs/langchain/langchain/llms/promptguard.py

```diff
@@ -0,0 +1,98 @@
+import logging
+from typing import Any, Dict, List, Optional
+
+from pydantic import Extra, root_validator
+
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+from langchain.prompts.base import StringPromptValue
+from langchain.utils import get_from_dict_or_env
+
+logger = logging.getLogger(__name__)
+
+
+class PromptGuardLLMWrapper(LLM):
+    """
+    A LLM that uses the PromptGuard library to sanitize and desanitize.
+
+
+    To use, you should have the `promptguard` python package installed,
+    and the environment variable `PROMPT_GUARD_ACCESS_TOKEN` set with
+    your access token, or pass it as a named parameter to the constructor.
+
+    Example:
+        .. code-block:: python
+
+            prompt_guard_llm = PromptGuardLLM(llm=ChatOpenAI())
+    """
+
+    llm: Any
+    """The LLM to use."""
+
+    class Config:
```

Comment:
What is this for?

Reply:
A configuration class for [Pydantic](https://docs.pydantic.dev/latest/)